### PR TITLE
update readme with usage

### PR DESCRIPTION
**Pull request by digitronik**
Signed-off-by: Nikhil Dhandre <nik.digitronik@live.com>

### CODE REVIEW

**Comment by digitronik** [comment]
@sbulage @ganeshhubale Need to update README before release.

File: README.md [hunk 1]
```diff
@@ -48,15 +48,13 @@ Simple command line application to spin [`selenium docker container`](https://hu
     
     - [Ubuntu](https://docs.docker.com/install/linux/docker-ce/ubuntu/) and [others](https://docs.docker.com/install/)
 
-2. **TigerVNC Versions(based on various Distro's)**:
+2. **TigerVNC Viewer**:
```
Comment (sbulage) [inline README.md:51]: :+1:

File: README.md [hunk 2]
```diff
@@ -67,28 +65,103 @@ or
 ```
 
 ## Usage:
-```
+
+```shell
+❯❯❯ miqsel --help
```
Comment (sbulage) [inline README.md:70]: **Required:**
Try to use `#` or `$` to denote shell commands.

**Review by sbulage — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -67,28 +65,103 @@ or
...
+    ❯❯❯ miqsel browser --firefox
+    Browser set to firefox
+    ❯❯❯ miqsel browser
+    firefox
+    ```
+
+- **Viewer**:
+Take `vnc` access of selenium server. At start of selenium server automatically `vnc` viewer will 
+`invoke`. If the `status` is `running` and the `viewer` is `close` then viewer help to reopen vnc session.
+
```
Comment (ganeshhubale) [inline README.md:147]: We can update above line like this
**Optional:**

At the start of selenium server, `vnc` viewer will be `invoked` automatically. If the `status` is `running` and  `viewer` is `closed` then ```viewer``` command will help to re-open vnc session.

File: README.md (hunk 2 above)
Comment (digitronik) [reply README.md:70]: @sbulage `$` indicate normal user and `#` indicate root user.... I think for python we wont check shell user.... it is applicable for both... you `>>>` is came from fish but its ok... looks great.

**Review by sbulage — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (digitronik) [reply README.md:147]: Added some changes....

**Pull request merged**